src/rofl/common/cpacket.h

```diff
@@ -77,7 +77,7 @@ class cpacket : public rofl::cmemory {
 	cpacket(
 			uint8_t *buf, size_t buflen, size_t head = DEFAULT_HSPACE, size_t tail = DEFAULT_TSPACE) :
 				rofl::cmemory(head + buflen + tail), head(head), tail(tail), initial_head(head), initial_tail(tail)
-		{ memcpy(somem()+head, buf, buflen); };
+		{ assert(buf); memcpy(somem()+head, buf, buflen); };
```

Comment:
headers for assert are not included, as travis says

https://travis-ci.org/bisdn/rofl-common/jobs/144980700#L1175